code/datums/manufacturing.dm

```diff
@@ -1537,6 +1537,16 @@
 	create = 1
 	category = "Component"
 	apply_material = TRUE
+	var/datum/forensic_id/roboprint = null // Give robo arms the same fingerprints
+
+	modify_output(var/obj/machinery/manufacturer/M, var/atom/A, var/list/materials)
+		..()
+		if(istype(A, /obj/item/parts/robot_parts/arm))
+			var/obj/item/parts/robot_parts/arm/new_arm = A
+			if(roboprint)
+				new_arm.limb_print = roboprint
+			else
+				roboprint = new_arm.limb_print
```

Comment:
Because blueprints are singletons, this makes all robot parts printed with this recipe have the same fingerprints, even across different fabs. IMO it's easiest to just let the fingerprints be different for the two cyborg arms, since printing the parts individually will have that behavior anyways.